log/term/terminal_windows.go

```diff
@@ -8,15 +8,23 @@
 package term
 
 import (
+	"bytes"
+	"encoding/binary"
 	"io"
+	"regexp"
 	"syscall"
 	"unsafe"
+
+	"golang.org/x/sys/windows"
```

Comment:
Done. I removed all the reference to external modules. I discovered that `syscall` actually already has a lot of useful stuff we needed, including for handling UTF16 names (on which Windows heavily relies).